content/get-involved.ts

```diff
@@ -77,27 +82,42 @@ const getInvolvedContent: GetInvolvedContent = {
     imgsrc: 'illustrations/climbing-guy.svg',
     mobileImgsrc: 'illustrations/checklist1.svg',
   },
-  volunteerPage: {
-    header: 'Help us break down barriers by joining the team.',
-    body: [
-      'You\'ll be a perfect fit if you:',
-    ],
-    facts: [
+  perfectFit: {
+    header: 'You\'ll Be Perfect If...',
+    body: 'You will thrive and feel connected within Clearviction if you embody the following key attributes and share these values...',
```

Comment:
maybe this could be `values:` instead i dont love the double ellipses in the header/body

Reply:
hey @emilyemilyemilyemilyemilyemily, I think it would be best to keep the `key` name as the element in the section. But tbh it's more of a header/subheader so maybe I should change it to something like `values-header` ? 
hahaha I totally missed that but agree about the double ellipses, I'll reach out to Dan

Reply:
I also agree about all of the `...` not being great, I think we should fix before merging.

Reply:
refactored the key name to be `valuesDescription`, waiting to hear back from designer on which ellipse to remove

Reply:
resolved